backend/store/playlist.go

```
@@ -0,0 +1,170 @@
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (ps PlaylistStore) ReorderSongsInPlaylist(playlistId uuid.UUID, songId uuid.UUID, newPosition int) error {
+	tx, err := ps.db.Beginx()
+	if err != nil {
+		return fmt.Errorf("failed to begin transaction: %w", err)
+	}
+	defer tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)